### DISCUSSION TITLE
nightDayRatio

### DISCUSSION THREAD

**Discussion by 9202014710** [mhm-ufz/mHM#78] [Q&A] [answered]
It's written "If read_meteo_weights is False than night fractions below are used" in mhm.nml (test_domian), but according to my test, there should be "If read_meteo_weights is True than night fractions below are used". Maybe it was a typo or something I misunderstood?

**Comment by pallavhydro** [comment, marked as answer]
Hi @9202014710 

The comment you pointed out in the mhm.nml file is correct and is not a typo. Following is the explanation:

The water balance in mHM runs internally at hourly time step. This requires mHM to have some basis to disaggregate the input daily forcings values (prec, pet, temp, ssrd, strd) to hourly values. There are currently two approaches in mhm for disaggregating the input daily forcings: First: `night-day ratios`, and second: `meteo_weights`. 

In the `night-day ratio` approach, user provides 12x night fractions for 12 months. As commented, the day fraction = 1 - night fraction. In this approach, all the day hours (6 am to 6 pm) disaggregated values are the same and all the night hours (6 pm to 6 am) disaggregated values are the same. Also, spatially, there is no variation of the fractions across domain grids/ cells.

The `meteo weights` approach provides more complexity. The user provides a netcdf file containing spatially varied, hourly weights for 12 months of a year. Thus, each hour of a day can have different value after disaggregation, which can be different at each cell. 

Hence, coming back to your question, when `read_meteo_weights` is set to `False`, mHM doesn't look for the meteo-weights file/s but rather uses the night ratios provided in mhm.nml.


I hope this explanation helps you in using mHM.

Good luck
Pallav

**Comment by 9202014710** [comment]
Thank you for your reply. I will do a test to verify it later.

**Reply by pallavhydro** [reply]
Hi @9202014710 

If the discussion here solved your issue, I'd appreciate you mark the response that solved the issue.

Thanks in advance.
Pallav
